lessons/Algebra2/exploring-quadratic-models/langs/en-us/pages/exploring-mpg.adoc

```diff
@@ -0,0 +1,78 @@
+
+@n Click "Run", and evaluate `mpg-table` in the Interactions Area. Do you know what each column means?
+
+[cols="1a,1a", grid="none", frame="none", stripes="none"]
+|===
+|
+- *test* - the number of the fuel efficiency test
+- *model* - the car used in the test (A-F)
+|
+- *speed* - the speed at which the car was tested
+- *mpg* - the miles-per-gallon the car achieved at that speed
+|===
+
+@n Evaluate `A15`, `A45` and `A75` in the Interactions Area. What *model* of car is used in all three rows? @fitb{}{@ifsoln{`A`}}
+
+@n At what *speeds* is this model being tested in each of the rows? @fitb{}{@ifsoln{15, 55, and 75 miles per hour}}
+
+@vspace{1ex}
+
+[.FillVerticalSpace, cols="1a,1a", options="frame",stripes="none"]
+!===
+| @n In Pyret, make a scatter plot showing the relationship between `speed` and `mpg`, using `model` as your labels, then sketch the resulting scatter plot below.
+|
+--
+Based on the scatter plot...
+
+@n What do you *Notice?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What do you *Wonder?* @fitb{}{} +
+@fitb{}{} +
+@fitb{}{}
+
+@n What speed is generally the most fuel-efficient? @fitb{}{@ifsoln{45-55mpg}}
+
+@n Which model is generally the most fuel-efficient? @fitb{}{@ifsoln{B}}
+
+@n Which model is generally the _least_ fuel-efficient? @fitb{}{@ifsoln{G}}
+--
+!===
+
+@n Do you see a correlation? Describe its form (linear/nonlinear/none), direction (positive/negative), and strength (strong/moderate/weak/none):
+
+@fitb{}{@ifsoln{There seems to be a somewhat strong, nonlinear correlation}}
+
+@fitb{}{@ifsoln{It looks like a parabola that opens down, with a dimple at the top}}
+
+@center{**Stop here! We'll continue after some discussion.**}
+
+@n A linear model is defined on line 22, with its Contract on line 21. What is its name? @fitb{2em}{@ifsoln{f}} Domain? @fitb{5em}{@ifsoln{Number}} Range? @fitb{}{@ifsoln{Number}}
+
+@n Fit the linear model `f` to the data using the `fit-model` function. What is the @math{R^2} value? @fitb{}{@ifsoln{-11.794}}.
+
+
+@n Experiment with the slope and y-intercept of `f` to improve the model. What is the @math{R^2}? @fitb{5em}{} Write the model below.
+
+@center{
+ @math{f(x) =} @fitbruby{8em}{}{slope}x + @fitbruby{8em}{}{y-intercept} @hspace{5em} @show{(code '(define (f x) (+ (* (?ANS varies) x) (?ANS varies))))}
+}
+
+@n Evaluate @show{(code '(lr-plot mpg-table "model" "speed" "mpg"))} to find the _best-possible_ linear model. What is the @math{R^2}? @fitb{5em}{@ifsoln{0.00426}}  Write the model below.
+
+@center{
+ @math{f(x) =} @fitbruby{8em}{@ifsoln{@math{-0.01017}}}{slope}x + @fitbruby{8em}{@ifsoln{29.9459}}{y-intercept} @hspace{5em} @show{(code '(define (f x) (+ (* (?ANS -0.01017) x) (?ANS 29.9459))))}
+}
+
+@fitb{}{}
+
+@n Could we get a better fit if we used a non-linear model? In your sketch (question 4, above), draw a line or curve that you think would fit better.
+
```

Comment:
Why ask them to draw a line if you've already asked about using a non-linear model?